complete/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
-#Thu Mar 01 09:04:46 CST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.2.1-bin.zip
```

Comment:
Why not upgrade to Gradle 5.6.3?

Reply:
5.2.1 is what I had installed. Spring Boot currently requires 4.10.

Reply:
Ahhh.... Well, the goal of the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html) is that you specify a version of Gradle used to build the project, regardless of what version of Gradle is installed on the user's machine. Then, when the user executes `./gradlew ...`, the wrapper will download the specified version of Gradle.

So, for a demo project, I pretty much always go with the latest and greatest version of Gradle, but in the end it's up to you.